flink-streaming-java/src/test/java/org/apache/flink/streaming/api/operators/async/AsyncWaitOperatorTest.java

```diff
@@ -1023,44 +999,6 @@ public void asyncInvoke(IN input, ResultFuture<OUT> resultFuture) throws Excepti
 		return in.transform("async wait operator", outTypeInfo, factory);
 	}
 
-	/**
-	 * Delay a while before async invocation to check whether end input waits for all elements finished or not.
-	 */
-	@Test
-	public void testEndInput() throws Exception {
```

Comment:
I am not sure whether we already have the tests to cover the case that `close()` process would wait for the inflight input finish. If not covered yet, we might refactor this test a bit for verifying that.

Reply:
That case has been covered by other tests, such as `AsyncWaitOperatorTest#testEventTime()`.